mkosi/__init__.py

```diff
@@ -3390,6 +3390,20 @@ def can_orphan_file(distribution: Optional[Distribution], release: Optional[str]
     ):
         env["SYSTEMD_REPART_MKFS_EXT4_OPTIONS"] = "-O ^orphan_file"
 
+    if ARG_DEBUG.get():
+        for dir in definitions:
+            for c in dir.glob("*.conf"):
+                logging.debug(f"\n === {c} ===")
+                with open(c) as f:
+                    # Do not spam the logs in case something goes wrong
+                    maxlines = 100
+                    for line in itertools.islice(f, maxlines):
+                        logging.debug(line.strip())
+
+                    for anything_left in f:
+                        logging.debug(f"=== {c.name} truncated to {maxlines} lines ===")
+                        break
```

Comment:
Let's shorten this somewhat, the truncation will never really be a thing
```suggestion
                logging.debug(f"# {c} (truncated to 100 lines)")
                with open(c) as f:
                    for line in itertools.islice(f, 100):
                        logging.debug(line.strip())
```
Using a `#` as the separator, makes these files still valid, when copied from screen.

Reply:
Done.